user_guide/data_management.rst

```diff
@@ -0,0 +1,234 @@
+----------------------------------------
+
+To change details of and existing record or create a new record, you need to (re)define the record properties on its 
+Create/Edit form.
+
+The Create/Edit form details are described in the :ref:`Create/Edit Forms `<user-guide-ui-components-create-pages>` 
+guide.
+
+Tag the Records
+---------------
+
+In order to give the system users additional information about the record, to systematize the records and mark them for 
+future usage (e.g. to select them for reports or create segments), you can tag them with existing or new tags. For 
+example, if you tag a customer as "VIP", this will let the other system users know about the customer's importance, you 
+can adjust report to show only the data for your VIPs, and make targeted mailings to them. Moreover, you can find a 
+record by its tag with the :ref:`search <user-guide-getting-started-search>` functionality.
+More information about tags is available in the :ref:`Tags <user-guide-ui-components-view-pages>` guide.
+ 
+
+Define the Process Within Which Record Details Can Be Changed And Follow It
+---------------------------------------------------------------------------
+
+In the process of the work with the customers is ongoing: carts turn into orders, 
+what seemed to be a potential opportunity appears not to be worth effort or tuns into a successful deal, and so on. 
+These and other processes are reflected in OroCRM with a change of record properties and sometimes creation of new 
+relations. (For example - a new lead appeared, and then it turned successful and was bound to some contact, some 
+opportunity and some customer).
+
+In many cases success of the business depends a lot on the unity of the procedures followed all through the company.
+For example, most companies won't allow their employees to close a cart without contacting the potential customer but
+no one wants to annoy the customers with duplicating calls. Customer complaints have to be researched and reacted to.
+Potential big contract requires some negotiations and only after that can turn into a successful deal or a lost chance.
+
+In order to regulate this, workflows can be created in OroCRM. Workflow defines what transitions (changes of 
+properties) are available to the system users at each step.
+
+You can find more details about workflows in the :ref:`Workflows <user-guide-workflow-management-basics>` guide.
+
+Log Actions in the System
+-------------------------
+
+Whether a workflow has been defined for entity records, or users can process it in any order, it is important to know 
+what actions have been performed with a record and by whom. Who has turned this cart into an order? Who has closed 
+an opportunity? Who has deleted a customer? Who has changed contact details? In order to know the answers, you 
+can use OroCRM's data audit functionality - with it you will see what actions have been performed in the system, when 
+and by whom.
```

Comment:
Comment: I deleted the last clause ("with it you will see what actions have been performed in the system, when 
 +and by whom") because the first sentence states the same information.